librz/bin/format/elf/elf_info.c

```diff
@@ -1124,6 +1141,8 @@ static Sdb *get_verdef_entry_sdb_aux(ELFOBJ *bin, Elf_(Verdef) * verdef_entry, s
 }
 
 static Sdb *get_verdef_entry_sdb(ELFOBJ *bin, Elf_(Verdef) * verdef_entry, size_t offset) {
+	Elf_(Verdaux) verdaux_entry;
```

Comment:
Better move these definitions to just before they are used for the first time. Here directly before `if (!get_verdaux_entry...` below.